client/marketing/coupons/style.scss

```diff
@@ -27,4 +27,31 @@
 		font-weight: normal;
 		margin-top: 4px;
 	}
+
+	.woocommerce-marketing-recommended-extensions-item {
+
+		.woocommerce-admin-marketing-product-icon {
+			transition: stroke 0.2s ease;
+			transition: fill 0.2s ease;
+			transition: background 0.2s ease;
+			overflow: hidden;
+			box-shadow: inset 0 0 0 1px #3858e9;
```

Comment:
This colour value is repeated 3 times, it should be a variable.
@j111q shouldn't this blue icon color be consistent with the rest of WP Admin? The new WC home screen actually matches the the current theme. E.g: 
![](https://d.pr/i/yxwsSu+)